docker/hive/Dockerfile

```diff
@@ -39,7 +39,93 @@ RUN git clone -v "https://github.com/kkrt-labs/kakarot-indexer.git"
 FROM mongo:6.0.8 as mongo
 
 # Ethereum RPC Server
-FROM ghcr.io/kkrt-labs/kakarot-rpc/node:v0.2.4 as rpc
+### Build the Cairo programs artifacts
```

Comment:
Weirdly this does not work anymore!

```
FROM ghcr.io/kkrt-labs/kakarot-rpc/node:v0.2.4 as rpc
``` 
Will allow hive tests to work locally,

But replacing it by the actual build will make it fail instantly

Reply:
For now I literally copied part of the RPC docker file to see if it worked

Reply:
I FIXED IT